### ISSUE TITLE
GitLab CI

### ISSUE THREAD

**Issue by ayufan** [danger/danger#1008]
Hi Everyone,

I wonder if we could make Danger for GitLab to use a JUnit backend for comments when used with CI.
We released the JUnit summary for Merge Requests: https://gitlab.com/gitlab-org/gitlab-ce/issues/45318.

For example, this message https://gitlab.com/gitlab-org/gitlab-ce/merge_requests/20576#note_88231832, could be translated into JUnit report that would be shown as part of the Merge Request widget, and thus in end would not pollute comments. We don't really yet have a notion of Warnings there, but it could be added if we consider this as a good extension.

**Comment by ayufan** [comment]
cc @rymai

**Comment by orta** [comment]
Yeah, I think this is a great call - I think making a different response platform and adding a way to let someone toggle between them is a great idea
